### PR TITLE
Powerstore, Powermax and their supported modules chart changes for CSM installation Wizard

### PR DESCRIPTION
**Pull request by chaganti-rajitha**
#### Is this a new chart?

There are changes in the existing charts and also new charts has been added.

#### What this PR does / why we need it:
As a part of CSM installation wizard, we need all charts to be placed at one place.

#### Which issue(s) is this PR associated with:
https://github.com/dell/csm/issues/591

#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Chart Version bumped
- [ ] Variables are documented in the chart README.md
- [ ] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

**Review by AkshaySainiDell — CHANGES_REQUESTED** [review]
For PowerStore min k8s version has been changed to 1.23. Please modify charts/csi-powerstore/Chart.yaml & charts/csi-powerstore/templates/_helpers.tpl accordingly,

**Review by AkshaySainiDell — APPROVED** [review]

**Review by boyamurthy — APPROVED** [review]
LGTM.

**Comment by tdawe** [comment]
Since these changes affect other helm charts, please get reviews from those teams as well including Novus for Authorization (@shaynafinocchiaro ), Yulan for Observability (@baoy1 ), and Phoenix for Replication (@harshaatdell )

**Review by baoy1 — APPROVED** [review]

File: charts/csm-authorization/values.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+  # base hostname for the ingress rules that expose the services
+  # the proxy-server ingress will use this hostname
+  # the tenant-service ingress will use tenant.hostname
+  # the role-service ingress will use role.hostname
+  hostname: csm-authorization.com
+
+  # log level for csm-authorization
+  logLevel: debug
+
```
Comment (shaynafinocchiaro) [inline charts/csm-authorization/values.yaml:29]: For Authorization, there has been a new parameter added https://github.com/dell/helm-charts/blob/release-1.5.1/charts/csm-authorization/values.yaml#L30.

File: charts/csm-authorization/values.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+redis:
+  images:
+    redis: redis:6.0.8-alpine
+    commander: rediscommander/redis-commander:latest  
+  # set the storageClass for redis to use. otherwise, the default storage class is used
+  # storageClass: local-storage
+  # Optionally, uncomment and specify the name of the pre-created namespace to install the redis in it
+  # namespace: 
+
+# Optionally, uncomment and specify the name of the pre-created namespace to install the module in it
```
Comment (shaynafinocchiaro) [inline charts/csm-authorization/values.yaml:92]: Duplicate comment here

File: charts/csm-authorization/values.yaml (hunk 1 above)
Comment (chaganti-rajitha) [reply charts/csm-authorization/values.yaml:29]: Thanks for reviewing, updated the charts with latest changes.

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by tdawe — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Review by santhoshatdell — APPROVED** [review]

**Pull request merged**